dialout/dialout_server_cli/dialout_server_cli.go

```diff
@@ -20,7 +19,7 @@ var (
 	caCert            = flag.String("ca_crt", "", "CA certificate for client certificate validation. Optional.")
 	serverCert        = flag.String("server_crt", "", "TLS server certificate")
 	serverKey         = flag.String("server_key", "", "TLS server private key")
-	insecure          = flag.Bool("insecure", false, "Skip providing TLS cert and key, for testing only!")
```

Comment:
Like client, you should keep this mode: InsecureSkipVerify

Reply:
Reverted and tls_disable has been added.